pyvistaqt/plotting.py

```diff
@@ -1,9 +1,43 @@
-"""Qt interactive plotter."""
+"""Qt interactive plotter.
+
+Inheritance
+===========
+
+BackgroundPlotter
++-- QtInteractor
+    |-- QVTKRenderWindowInteractor
+    |   +-- QWidget
+    +-- BasePlotter
+
+MainWindow
++-- QMainWindow
```

Comment:
Thanks, this was needed.

Reply:
It'd be good to autodoc this in the docs. @larsoner, could you pull the changes on master and add this somewhere in the API reference page with an autodoc